### PR TITLE
LPS-147943 | master

### PR DESCRIPTION
**Pull request by ces-sontruong**
@hongvo2308 Please help me review this PR and give me feedback. Thank you!
Ticket: [LPS-147943](https://issues.liferay.com/browse/LPS-147943)

### CODE REVIEW

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const ResultsBar = ({
 					<span className="component-text text-truncate-inline">
 						<span className="text-truncate">
 							{Liferay.Util.sub(
-								Liferay.Language.get('x-results-for-x'),
+								(itemsTotal === 1) ? Liferay.Language.get('x-result-for-x') : Liferay.Language.get('x-results-for-x'),
```
Comment (ces-hongvo) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js]: @sontruongces 
Can we try to use Liferay.Language.get((itemsTotal === 1) ?'x-result-for-x':'x-results-for-x')

**Review by hongvo2308 — CHANGES_REQUESTED** [review]

**Comment by ces-sontruong** [comment]
@hongvo2308 I have just rebased and updated the fix. Please help me to review it again. Thank you so much!

**Comment by ces-hongvo** [comment]
@sontruongces Please rebase the upstream/master and resolve the confilct
Thanks.

**Comment by ces-hongvo** [comment]
Forwarded https://github.com/liferay-ts-reviewer/liferay-portal/pull/36
Thank you.

**Pull request closed without merging**